tls/s2n_ecc_preferences.c

```diff
@@ -33,6 +33,12 @@ const struct s2n_ecc_named_curve *const s2n_ecc_pref_list_20200310[] = {
     &s2n_ecc_curve_secp384r1,
 };
 
+const struct s2n_ecc_named_curve *const s2n_ecc_pref_list_20201010[] = {
```

Comment:
is there a reason why you excluded 25519 in this preference list ?

Reply:
no specific reason, but the potential usage for us might be only need those 3 curve(521r1, 384r1, 256r1)

Reply:
will add a new curve preference to include all curve, and leave this 20201010 preference for our future usage